clevis/Sanity/password-generation/test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+PACKAGES="clevis clevis-luks tang jose luksmeta cryptsetup"
```

Comment:
Where is this "PACKAGES" variable used?

Reply:
This is very unintuitive. `rlAssertRpm --all` uses it [1].

[1] https://www.mankier.com/1/beakerlib#Functions-Rpm_Handling